js/card.js

```diff
@@ -0,0 +1,100 @@
+"use strict";
+(function () {
+
+  const ENGLISH_NAME_ROOM_TO_RUSSIAN_NAME_ROOM = {
+    flat: "Квартира",
+    bungalow: "Бунгало",
+    house: "Дом",
+    place: "Дворец"
+  };
```

Comment:
перечесление записывается не так, да и название переменной не очень
```suggestion
  const TypeTitle = {
    FLAT: "Квартира",
    BUNGALOW: "Бунгало",
    HOUSE: "Дом",
    PLACE: "Дворец"
  };
```